packages/platform-browser/src/hydration.ts

```diff
@@ -100,8 +100,18 @@ export function withI18nSupport(): HydrationFeature<HydrationFeatureKind.I18nSup
 /**
  * Enables support for event replay
  *
+ * @usageNotes
+ *
+ * Basic example of how you can enable event replay in your application when
+ * `bootstrapApplication` function is used:
+ * ```
+ * bootstrapApplication(AppComponent, {
+ *   providers: [provideClientHydration(withEventReplay)]
```

Comment:
```suggestion
 *   providers: [provideClientHydration(withEventReplay())]
```

Shouldn't this be called?

Reply:
@eneajaho thanks for catching this, we've updated the docs.